custom_components/trakt_tv/models/kind.py

```diff
@@ -12,6 +12,13 @@ class CalendarInformation:
     model: Media
 
 
+@dataclass
+class ListInformation:
+    identifier: str
+    name: str
```

Comment:
I prefere you to remove it and to handle the possible None value in sensors:
```

@property
    def name(self):
````

Otherwise, your sensor will be named "Trak List Name " instead of "Trak List Name"